src/component/admin/apiToken/ApiTokenForm/SelectProjectInput.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { Fragment, FC, useMemo, useState } from 'react';
+import {
+    Link,
+    Checkbox,
+    FormControlLabel,
+    TextField,
+    Box,
+    Paper,
+} from '@material-ui/core';
+import { Autocomplete } from '@material-ui/lab';
+import CheckBoxOutlineBlankIcon from '@material-ui/icons/CheckBoxOutlineBlank';
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+import { IAutocompleteBoxOption } from 'component/common/AutocompleteBox/AutocompleteBox';
+import { useStyles } from './ApiTokenForm.styles';
+
+const ALL_PROJECTS = '*';
+
+const icon = <CheckBoxOutlineBlankIcon fontSize="small" />;
+const checkedIcon = <CheckBoxIcon fontSize="small" />;
+
+// Fix for shadow under Autocomplete - match with Select input
+const CustomPaper: FC = ({ ...props }) => <Paper elevation={8} {...props} />;
+
+const SelectProjectInput: FC<{
```

Comment:
Could just type the props directly here instead of using FC, just since there's no `children`.

Reply:
Should we maybe use `VCP`? It's more consistent to have types always first.